src/misc/cache/lv_image_cache.h

```diff
@@ -15,6 +15,8 @@ extern "C" {
  *********************/
 #include "lv_cache_private.h"
```

Comment:
Can this be moved to the .c file? https://github.com/lvgl/lvgl/issues/5899#issuecomment-2004582233

Reply:
`<stdbool.h>` will be needed here fyi, if you do move it.